arangod/Aql/Functions.cpp

```diff
@@ -2847,6 +2847,57 @@ AqlValue Functions::DateDaysInMonth(arangodb::aql::Query* query,
   return AqlValue(AqlValueHintUInt(static_cast<uint64_t>(unsigned(lastMonthDay.day()))));
 }
 
+/// @brief function DATE_TRUNC
+AqlValue Functions::DateTrunc(arangodb::aql::Query* query,
+                            transaction::Methods* trx,
+                            VPackFunctionParameters const& parameters) {
+  static char const* AFN = "DATE_TRUNC";
+  using namespace std::chrono;
```

Comment:
date and std::chrono implement same types with upcoming standards. We can avoid collisions now!